src/main/java/org/metricshub/wmi/wbem/WmiWbemServices.java

```diff
@@ -1038,7 +1046,8 @@ public void objectPut(
 							pWbemClassObject,
 							wszName,
 							0, // lFlags -- Reserved, must be zero
-							pVal},
+							pVal,
+							0},
```

Comment:
Please add a comment here along with the link to the official documentation: https://learn.microsoft.com/en-us/windows/win32/api/wbemcli/nf-wbemcli-iwbemclassobject-put